lib/windowTailor.js

```diff
@@ -0,0 +1,75 @@
+/** Used to calculate browser dimensions based on PDF size */
+const HTML_DPI = 96
```

Comment:
It might be helpful to define DPI in the comment, in case someone isn't familiar with that acronym.